lib/support/browserScript.js

```diff
@@ -46,6 +47,54 @@ async function getScriptsForCategories(categories) {
   return result;
 }
 
+/**
+ * Take the name of a script, its location on the disk
+ * and its contents and generate a ScriptObject:
+ *
+ * name: The name of the script.
+ * function: A function object if this is a new-style
```

Comment:
uber nit: this all seems too horizontally narrow.  This is <60 chars; I'm sitting here able to view 4k.

Reply:
Show off.